CONTRIBUTING.rst

```diff
@@ -0,0 +1,31 @@
+==============
+ Contributing
+==============
+Nice having you onboard!
+
+We love contributions from everyone and here are a few things to keep in mind when contributing.
+
+Issues
+======
+Somethings not working like it should? We've got you covered!
+
+ * Use github to create an issue;
+ * Write a descriptive title;
+ * Fill out the issue template;
+ * Wait for a developer to make coffee and get back to you;
+
+Pull requests
+=============
+You want to help out by contributing code? Here's a checklist for a pull request:
+
+ * Fork the project;
+ * Create a branch for your code;
+ * Write your awesome code;
+ * Adhere to the coding style guide {insert link to styleguide here};
```

Comment:
@RosienVT: We moeten een style guide hebben. Bij voorkeur een waarnaar we integraal kunnen verwijzen :)